subsys/mpsl/mpsl_init.c

```diff
@@ -19,6 +19,12 @@
 
 LOG_MODULE_REGISTER(mpsl_init, CONFIG_MPSL_LOG_LEVEL);
 
+/* The following two constants are used in nrfx_glue.h for marking these PPI
+ * channels and groups as occupied and thus unavailable to other modules.
+ */
+const uint32_t z_mpsl_used_nrf_ppi_channels = MPSL_RESERVED_PPI_CHANNELS;
+const uint32_t z_mpsl_used_nrf_ppi_groups;
+
```

Comment:
These are still called z_bt_ctlr_* in nrfx.glue.h. Shouldn't there be an update to nrfx together with this change?

Reply:
That's right. In [pr#80](https://github.com/zephyrproject-rtos/hal_nordic/pull/80) I combine the use of `z_mpsl_used_nrf_ppi_*` with the already used `z_bt_ctlr_used_nrf_ppi_*`.

For clarification - my intention is to prevent these PPI runtime allocation (through nrfx) in case where bt_ctlr is disabled but mpsl is enabled (e.g. only for FEM support)

Reply:
Understood. I was just expecting that update to nrfx to be included in this PR. But I can see that the order is right, so either is fine.